pages/learn/manage/apps.md

```diff
@@ -0,0 +1,137 @@
+    data:
+      url: 'https://github.com/balenalabs/balena-dash'
+  logo:
+    type: blob.asset
+    data:
+      url: >-
+        https://raw.githubusercontent.com/balenalabs/balena-dash/master/assets/logo.png
+data:
+  applicationConfigVariables:
+    - BALENA_HOST_CONFIG_gpu_mem: 128
+    - BALENA_HOST_CONFIG_dtoverlay: "vc4-fkms-v3d"
+  applicationEnvironmentVariables:
+    - SHOW_CURSOR: 0
+    - CONTROL_TV: 0
+    - ENABLE_GPU: 0
+    - KIOSK: 1
+    - PERSISTENT: 1
+    - PORTAL_SSID: balenaDash
+    - PORTAL_PASSPHRASE: balenaDash
+    - ACTIVITY_TIMEOUT: 600
+    - LAUNCH_URL: ''
+    - ROTATE_DISPLAY: normal
+    - WINDOW_POSITION: '0,0'
+    - LOCAL_HTTP_DELAY: 0
+    - FBCP_DISPLAY: ''
+  defaultDeviceType: raspberrypi4-64
+  supportedDeviceTypes:
+    - raspberrypi4-64
+    - fincm3
+    - raspberrypi3
+    - raspberrypi3-64
+    - intel-nuc
+    - genericx86-64-ext
+    - raspberrypi400-64
+version: 2.0.0
+```
+
+Next, navigate to the Apps tab in the sidebar and click the `Create App` button in the balenaCloud dashboard.
+
+Fill the details for your App in the modal, and click `Create new app` button to create your App.
+
+Next, we have to push a new release to your newly created App on balenaCloud. If you are new to this, then follow the Getting Started guide.
+
+## Sharing an App on balenaHub
+
+By default, Apps that you create within the balenaCloud dashboard are private and only accessible by the owner along with any other teams or members that you explicitly grant access to. When you are ready to release, head to the Settings tab in the sidebar and add your App’s GitHub repository to the Repository URL section. Next, toggle the App visibility button to on for your App to be visible on balenaHub. You’re free to toggle the visibility on and off as necessary at any time.
+
+![Enable App visibility on balenaHub](/img/common/app/visibility-toggle.png)
+
+Apps marked with visibility set to ‘on’ in the dashboard will be made available as another resource for the community on balenaHub. Users can deploy Apps from balenaHub to their own fleets managed in their own balenaCloud accounts using the `Deploy` button with each App. 
+
+
+## Deploying Apps from balenaHub
+
+![Deploy an App from balenaHub](/img/common/app/deploy-from-balenahub.png)
+
+When the deploy button on balenaHub is used, the balenaCloud builder will fetch the latest source from the linked GitHub repository and create a new build for the chosen fleet. This process uses the [Deploy with balena](https://www.balena.io/docs/learn/deploy/deploy-with-balena-button/) workflow in order to guide users through the process of creating/signing in to a balenaCloud account, creating a fleet and deploying the App.
+
+
+## Support access
+
+If you require assistance with an App and have contacted us for support, agents may ask you to ‘enable support access’. This can be achieved via toggling the `Support access` setting, found within the settings page for your App, to on. 
```

Comment:
```suggestion
If you require assistance with an App and have contacted balena for support, agents may ask you to ‘enable support access’ for the App. This can be achieved via toggling the `Support access` setting, found within the settings page to on. 
```